src/provider/model/FinEnvironment.ts

```diff
@@ -93,10 +93,10 @@ export class FinEnvironment extends AsyncInit implements Environment {
             const application = fin.Application.wrapSync(identity);
             const applicationInfo = await application.getInfo();
 
-            const {shortcut, startup_app} = applicationInfo.manifest as OFManifest;
+            const {shortcut, startup_app} = (applicationInfo.manifest || {}) as OFManifest;
```

Comment:
If the `ApplicationInfo` interface is inaccurate, and `manifest` can be null/undefined, would comment that here (with another TODO referencing 840 if we expect the types to be updated soon).

Would also change the `OFManifest` interface, so we have `startup_app?` rather than `startup_app`, otherwise casting `{}` to `OFManifest` is potentially misleading